base_layer/core/src/transactions/transaction/transaction_input.rs

```diff
@@ -353,7 +345,6 @@ impl TransactionInput {
                 .chain(self.script_signature.public_nonce().to_consensus_bytes())
```

Comment:
you don't have to split the comsig here anymore. You have added an implementation to consensus encode the entire comsig.